ffi/diplomat/src/lib.rs

```diff
@@ -7,10 +7,16 @@
         all(feature = "freertos", not(feature = "x86tiny")),
         all(feature = "x86tiny", not(feature = "freertos")),
     ),
-    feature(alloc_error_handler)
+    feature(alloc_error_handler),
+    no_std,
+    deny(
+        clippy::indexing_slicing,
+        clippy::unwrap_used,
+        clippy::expect_used,
+        clippy::panic,
+        clippy::upper_case_acronyms
+    )
 )]
-#![no_std]
```

Comment:
Issue: This is not correct. The `feature(alloc_error_handler)` should be enabled in some cases, but `#![no_std]` and `#![allow(clippy:upper_case_acronyms)]` should always be enabled. Please revert this back to what it was before and add the new clippy exclusions as a standalone cfg_attr.

Reply:
Issue: This is still not correct. You should have

```
#![no_std]
#![allow(clippy::upper_case_acronyms)]
#![cfg_attr(
    not(test),
    deny(
        clippy::indexing_slicing,
        clippy::unwrap_used,
        clippy::expect_used,
        clippy::panic
    )
)]
```